src/utils/test/testThunkConformance.js

```diff
@@ -57,8 +74,8 @@ function mockResolve() {
 }
 function doMockAxiosRequests(promiseFn) {
   const mockFn = jest.fn(url => promiseFn());
-  Axios.get = mockFn;
-  Axios.put = mockFn;
-  Axios.post = mockFn;
-  Axios.delete = mockFn;
+  axios.get = mockFn;
+  axios.put = mockFn;
+  axios.post = mockFn;
+  axios.delete = mockFn;
```

Comment:
E det her i bruk?

Reply:
Ja. Må vær der for å ikkje sende faktiske requests i testane

Reply:
Den er grei